### ISSUE TITLE
Support JSON-LD 1.1

### ISSUE THREAD

**Issue by miguel76** [digitalbazaar/jsonld.js#203]
Add support for the new features introduced in JSON-LD 1.1

**Comment by stain** [comment]
In particular this example with [localized](https://json-ld.org/spec/latest/json-ld/#scoped-contexts) `@vocab`/`@context` seems to fail:

```json
{ "@context": {    
  "oa": "http://www.w3.org/ns/oa#",
  "link": "https://tools.ietf.org/vocab/rfc5988#",  
  "rel": {
    "@id": "link:rel",
    "@type": "@vocab",
    "@context": {"@vocab": "https://www.iana.org/assignments/link-relations/"}
  },
  "anchor": {"@id": "link:anchor", "@type": "@id"},
  "href": {"@id": "link:href", "@type": "@id"},
  "title": {
      "@id": "http://purl.org/dc/terms/title"
   }  
 },
 "@graph": [
  
   {"href"   : "http://example.com/foo",
    "title": "Fred",
    "meh": "fred",
    "anchor" : "http://example.net/bar",
    "rel" : [ "next" , "http://example.net/linkrel" ] }
  ] 
}
```

http://rdf.greggkellogg.net/distiller will correctly honour the inner `@context` and outputs:

```turtle
[]
   <https://tools.ietf.org/vocab/rfc5988#rel> <https://www.iana.org/assignments/link-relations/next>,
       <http://example.net/linkrel> .
```

**Comment by CumpsD** [comment]
`"@container": ["@set", "@language"],` seems to fail as well:

Spec:
https://github.com/json-ld/json-ld.org/issues/407
https://github.com/json-ld/json-ld.org/pull/503

Example: http://tinyurl.com/y8ejvou9

**Comment by davidlehn** [comment]
At the moment none of 1.1 has been implemented due to lack of time and resources.  We'll get to 1.1 as soon as we can.  We want to use the fancy new features too!  Any help would be appreciated.  The 0.5.x branch has progress on other issues.

**Comment by davidlehn** [comment]
1.0.0 released supporting 1.1 draft features.
See https://github.com/digitalbazaar/jsonld.js#conformance.
